### ISSUE TITLE
tasks/emoji.rake does not seem to exist

### ISSUE THREAD

**Issue by mikemclaren** [sofetch/slack-standup-bot#53]
This might just be a misunderstanding on my part, but when I run the rake command while following the steps you've outlined, I'm getting the following error:

```
slack-standup-bot [master] rake db:create db:migrate
rake aborted!
LoadError: cannot load such file -- tasks/emoji.rake
REDACTED_PATH/slack-standup-bot/Rakefile:4:in `load'
REDACTED_PATH/slack-standup-bot/Rakefile:4:in `<top (required)>'
(See full trace by running task with --trace)
```

Apologies if I'm just overlooking something obvious!

**Comment by santiagodoldan** [comment]
Hello mikemclaren,

Did you install the gems? it seems that the "gemoji" gem is not installed.

Just to be sure that's the error, what's the output of this command? -> gem list | grep gemoji

Cheers

**Comment by mikemclaren** [comment]
```

**Comment by santiagodoldan** [comment]
mmmm, that's odd, I removed that gem and got the same error as you.

ruby/slack-standup-bot [---●] » rake db:migrate
WARNING: Nokogiri was built against LibXML version 2.9.0, but has dynamically loaded 2.9.2
I, [2016-01-21T11:05:04.915075 #24356]  INFO -- : Celluloid 0.17.2 is running in BACKPORTED mode. [ http://git.io/vJf3J ]
Model files unchanged.
ruby/slack-standup-bot [---●] » gem uninstall gemoji
Successfully uninstalled gemoji-2.1.0
ruby/slack-standup-bot [---●] » rake db:migrate
rake aborted!
LoadError: cannot load such file -- tasks/emoji.rake
---/slack-standup-bot/Rakefile:4:in `load'
---/slack-standup-bot/Rakefile:4:in `<top (required)>'
/Users/---/.rvm/gems/ruby-2.0.0-p598@standupbot/bin/ruby_executable_hooks:15:in `eval'
/Users/---/.rvm/gems/ruby-2.0.0-p598@standupbot/bin/ruby_executable_hooks:15:in `<main>'
(See full trace by running task with --trace)

are you able to execute this task "rake emoji"?

**Comment by mikemclaren** [comment]
Ran `rake emoji` and got the same error as before.

**Comment by santiagodoldan** [comment]
are you using rbenv or rvm? try to add bundle exec before the command, bundle exec rake db:migrate db:create

**Comment by mikemclaren** [comment]
Running bundle exec caused some other errors to fire, and after some examining, the culprit seemed to be nokogiri (surprise, surprise), and that build error got buried along the way. I finally got it to install properly, but I needed to update nokogiri to 1.6.7 to get it to install.

But after that, I was able to get everything up and running properly! Thanks for the help! Excited to try this out.
